### ISSUE TITLE
Automatically follow console text

### ISSUE THREAD

**Issue by KoBeWi** [godotengine/godot-proposals#666]
**Describe the project you are working on:**
Game made in Godot.

**Describe the problem or limitation you are having in your project:**
I sometimes do a lot of per-frame printing. By default the output (I mean the one in the editor) follows what is printed, so you can observe how the values change over time. This effect stops when you scroll the console log. It's useful, because you can read the history while the values are still being printed.

My problem is that the following stops permanently. To "enable" it again, you need to scroll the log to bottom, so that it again starts following. It's a problem when after stopping project I read my values and then forget to scroll down, so I have to do it during runtime, which sometimes means I have to restart the game again, because I missed something. It's really annoying.

**Describe the feature / enhancement and how it helps to overcome the problem or limitation:**
Make the console log follow the output automatically when the project is started. This way ppl won't need to remember to scroll down each time before running the game.

**Describe how your proposal will work, with code, pseudocode, mockups, and/or diagrams:**
Basically what I wrote above.

**If this enhancement will not be used often, can it be worked around with a few lines of script?:**
It's editor thing.

**Is there a reason why this should be core and not an add-on in the asset library?:**
Editor. Thing.

**Comment by Calinou** [comment]
Related to https://github.com/godotengine/godot/issues/37476.
